### PR TITLE
Improved documentation for abi3

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ PyO3 versions, please see the [migration guide](https://pyo3.rs/master/migration
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- Add support for building for CPython limited API. [#1152](https://github.com/PyO3/pyo3/pull/1152)
+
+### Changed
```
Comment (davidhewitt) [inline CHANGELOG.md:12]: I would also like to see the point about requiring `#[pyclass(subclass)]` here, as I think it's potentially quite breaking to some user code:

```suggestion
### Changed
- `#[pyclass(subclass)]` is now required for subclassing from Rust (was previously just required for subclassing from Python). [#1152](https://github.com/PyO3/pyo3/pull/1152)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ PyO3 versions, please see the [migration guide](https://pyo3.rs/master/migration
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- Add support for building for CPython limited API. [#1152](https://github.com/PyO3/pyo3/pull/1152)
```
Comment (davidhewitt) [inline CHANGELOG.md:10]: I think the other sentence can be added here:

```suggestion
### Added
- Add support for building for CPython limited API. This required a few minor changes to runtime behaviour of of pyo3 `#[pyclass]` types. See the migration guide for full details. [#1152](https://github.com/PyO3/pyo3/pull/1152)
```

**Review by davidhewitt — COMMENTED** [review]
Thanks, a few quick refinements to this and then let's merge it into the `abi3` branch!

**Comment by alex** [comment]
Changes incorporated. Thanks!

**Review by davidhewitt — APPROVED** [review]
Thanks!

**Comment by alex** [comment]
Thank you!

**Pull request merged**